dnf-behave-tests/dnf/plugins-core/copr.feature

```diff
@@ -136,7 +136,13 @@ Given I create and substitute file "//etc/dnf/plugins/copr.conf" with
 
       Please do not file bug reports about these packages in Fedora
       Bugzilla. In case of problems, contact the owner of this repository.
-      Error: This repository does not have any builds yet so you cannot enable it now.
```

Comment:
This has changed in Copr. The repository can be enabled even if there is no build yet, you just have to choose the right repository.